### PR TITLE
Add jrumble@1.3.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10335

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): 
 * Official website (optional, not the repository): https://github.com/jackrugile/jrumble/
 * License and its reference: [MIT](https://github.com/jackrugile/jrumble/blob/master/MIT-License.txt)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 21
   - Count of stars: 389
   - Count of forks: 62

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

Note https://www.npmjs.com/package/jrumble is not created by the author.

### CODE REVIEW

**Comment by extend1994** [comment]
@cdnjs/intern3 please help me review this pull request, thank you.

File: ajax/libs/jrumble/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "keywords": [
+    "jquery",
+    "rumble",
+    "vibrate",
+    "shake"
+  ],
+  "author": "Jack Rugile <jack@jackrugile.com>",
+  "license": "MIT",
+  "homepage": "http://jackrugile.com/jrumble",
```
Comment (sufuf3) [inline ajax/libs/jrumble/package.json]: https

**Comment by extend1994** [comment]
@sufuf3 I modified the `homepage`, please help me review it again, thank you.

**Comment by extend1994** [comment]
@PeterDaveHello Please help me review this pull request, thank you.

**Pull request merged**